src/debugAdapter/goDebug.ts

```diff
@@ -1587,6 +1585,75 @@ class GoDebugSession extends LoggingDebugSession {
 		});
 	}
 
+	private logDelveError(err: any, message: string) {
+		if (err) {
```

Comment:
Either make an early exit if `err` does not exist or we can make the `if (err)` check before calling `logDelveError`